### PR TITLE
Update `TestContainers` dependency

### LINKED ISSUE #400: Update testcontainers
Update testcontainers after removing old connector from dependencies https://github.com/tarantool/cartridge-java-testcontainers/issues/75

### PR DESCRIPTION
**Pull request by nickkkccc**
haven't forgotten about:
- [x] Tests
- [x] Changelog
- [ ] Documentation
- [x] Commit messages comply with the [guideline](https://www.tarantool.io/en/doc/latest/dev_guide/developer_guidelines/#how-to-write-a-commit-message)
- [x] Cleanup the code for review. See [checklist](https://github.com/tarantool/cartridge-java/blob/master/docs/review-checklist.md)

Related issues:
Closes #400.

### CODE REVIEW

File: src/test/java/io/tarantool/driver/core/proxy/CRUDOperationOptionsTest.java [hunk 1]
```diff
@@ -1,70 +0,0 @@
-package io.tarantool.driver.core.proxy;
```
Comment (iDneprov) [inline src/test/java/io/tarantool/driver/core/proxy/CRUDOperationOptionsTest.java:1]: Why did you delete that test?

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,17 +2,15 @@
...
-
-- Add `"mode"` option for select operation ([#107](https://github.com/tarantool/cartridge-java/issues/107))
-- Change using of proxy client parameters (`mode`, `rollback_on_error`, `stop_on_error`) with enum classes ([#419](https://github.com/tarantool/cartridge-java/issues/419))
-
 ### Bugfixes
 
 - Fix Instant converter to parse 8 bytes datetime ([#408](https://github.com/tarantool/cartridge-java/issues/408))
 
-### Internal changes
+### Internal and API changes
```
Comment (iDneprov) [inline CHANGELOG.md:9]: Why did you decide to put this together?

File: src/test/java/io/tarantool/driver/core/proxy/CRUDOperationOptionsTest.java (hunk 1 above)
Comment (nickkkccc) [reply src/test/java/io/tarantool/driver/core/proxy/CRUDOperationOptionsTest.java:1]: It's the same as #427 , just so you don't have to rebase when merging.

File: CHANGELOG.md (hunk 2 above)
Comment (nickkkccc) [reply CHANGELOG.md:9]: I did it similar to the titles introduced earlier so as not to break the style.

**Review by iDneprov — APPROVED** [review]
